### ISSUE TITLE
[Logging v2] Add logging upstream 3.8.4 chart

### ISSUE THREAD

**Issue by nickgerace** [rancher/rancher#30750]
Releases: https://github.com/banzaicloud/logging-operator/releases

**Comment by nickgerace** [comment]
PRs for this issue are pending on the following issues being moved to test.
- [ ] #30329 - partial fix
- [ ] #30595 - full fix
- [ ] #30383 - full fix

**Comment by nickgerace** [comment]
Would prefer to wait for the following change to be included in a release: https://github.com/banzaicloud/logging-operator/pull/676

**Comment by nickgerace** [comment]
Chart 3.8.4 has been removed from upstream. Closing in favor of https://github.com/rancher/rancher/issues/30917
